process/interface.go

```diff
@@ -675,11 +675,13 @@ type NetworkConnectionWatcher interface {
 
 // SCQuery represents a prepared query for executing a function of the smart contract
 type SCQuery struct {
```

Comment:
this is duplicated with the one from proxy. 
can we reuse the same structure? (e.g import from core?)